lua/dy.lua

```diff
@@ -0,0 +1,31 @@
+local dylib = require("dylib")
```

Comment:
The name of this file should be `example_of_dynamic_config_request.lua` to make clear that it's just an example, not part of the configuration itself.